modules/nupac.nu

```diff
@@ -367,6 +411,5 @@ export def "nupac upgrade" [
         error make {
           msg: "Either a list of packages or --all flag must be provided"
         }
-        exit 1
```

Comment:
`exit 1`s was deleted because it kills the current shell, making the error message useless. we just need to wait for the 0.65